packages/backend/src/clipboard/item-converters.ts

```
@@ -0,0 +1,158 @@
+import type {Stats} from "node:fs";
+
+type GetBaseProps = {
+    hash: ClipboardItemHash
+}
+const getBase = ({hash}: GetBaseProps): ClipboardBaseItem => ({
+    id: nanoid(),
+    hash,
+    dateTimeCreated: Date.now(),
+});
+
+type ConverterType<Base extends Object> = Base & {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)